runtime/westend/src/lib.rs

```diff
@@ -878,6 +879,72 @@ impl xcm_executor::Config for XcmConfig {
 	type ResponseHandler = ();
 }
 
+/// Type to convert an `Origin` type value into a `MultiLocation` value which represents an interior location
+/// of this chain.
+pub type LocalOriginToLocation = (
+	// And a usual Signed origin to be used in XCM as a corresponding AccountId32
+	SignedToAccountId32<Origin, AccountId, WestendNetwork>,
+);
+
+pub struct OnlyWithdrawTeleportForAccounts;
+impl frame_support::traits::Contains<(MultiLocation, Xcm<Call>)> for OnlyWithdrawTeleportForAccounts {
+	fn contains((ref origin, ref msg): &(MultiLocation, Xcm<Call>)) -> bool {
+		use xcm::v0::{
+			Xcm::WithdrawAsset, Order::{BuyExecution, InitiateTeleport, DepositAsset},
+			MultiAsset::{All, ConcreteFungible}, Junction::{AccountId32, Parachain},
+			MultiLocation::{Null, X1},
+		};
+		match origin {
+			// Root is allowed to execute anything.
+			Null => true,
```

Comment:
It seems that this will mean we cannot test plurality XCMs on Westend?

Reply:
We don't have a council on Westend since we have sudo. Your point might still be relevant though.

Reply:
We could introduce a council on Westend, but I think it's probably enough to test on Rococo.